### PR TITLE
Add workflow files

### PR DESCRIPTION
**Pull request by dangershony**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    - name: Release
+      uses: sondreb/action-release@main
+      with:
+        commit: 'master'
+        token: ${{ secrets.GITHUB_TOKEN }}
+        files: "${{env.RELEASE_PATH}}"
+        draft: true
+        prerelease: false
+        body: ''
+        name: "Blockcore Dns (Release ${{env.VERSION}})"
```
Comment (sondreb) [inline .github/workflows/build.yml:121]: Is name "Blockcore DNS" or "Blockcore Dns"?

File: src/Blockcore.Dns/Dockerfile.Release [hunk 2]
```diff
@@ -0,0 +1,13 @@
+FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-buster-slim
+WORKDIR /usr/local/app/
+
+RUN apt-get update \
+    && apt-get install -y curl \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN curl -Ls https://github.com/block-core/blockcore-dns/releases/download/#{VERSION}#/Blockcore.dns-#{VERSION}#-linux-x64.tar.gz \
```
Comment (sondreb) [inline src/Blockcore.Dns/Dockerfile.Release:9]: "Blockcore.dns", is that the same as output from build? Should be correct case at least.

Comment (dangershony) [reply src/Blockcore.Dns/Dockerfile.Release:9]: I sort of just did similar to the indexer project

File: .github/workflows/build.yml (hunk 1 above)
Comment (dangershony) [reply .github/workflows/build.yml:121]: I suppose for consistency `Dns`

**Pull request merged**